SRC/seleniumdemotest.py

```diff
@@ -1,48 +1,68 @@
 from selenium import webdriver
 import time
 import pytest
+from selenium.common import NoSuchElementException
 from selenium.webdriver.common.by import By
 from selenium.webdriver.support import expected_conditions
 from selenium.webdriver.common.keys import Keys
 import random
+WEBSITE_LINK = "http://seleniumdemo.com/?page_id=7"
+CORRECT_PASSWORD = "#1QAZwsx#1"
+WRONG_PASSWORD = "#1QAZwsx#111"
+CORRECT_LOGIN = ""
```

Comment:
this is not a final variable if you assign it in a following test

Reply:
that leads me to another thing the unit tests should not be dependent on other unit tests. in your case if the registration test will fail the login test would fail as well not because there is something wrong with loging mechanism but because the user was not created it should not be like that. what if you change the order of test execution?? what if you want to execute just one of those tests ?